stdgo/reflect/Reflect.hx

```diff
@@ -25,63 +25,14 @@ enum GT_enum {
 	GT_string;
 	GT_unsafePointer;
 	GT_chan(elem:GT_enum);
-	GT_interface;
+	GT_interface(name:String,type:GT_enum);
```

Comment:
The only type info that GT_interface could sensibly have is an array of GT_enum that are function declarations defining what the interface is.

Reply:
This would be sensible, because classes need to be assessed at runtime to see if they comply with the interface. This is particularly true of native Haxe classes passed to Go2hx libraries, where no `implements some_go2hx_interface` would be present.